app/lib/features/chat/widgets/rooms_list.dart

```diff
@@ -276,14 +277,10 @@ class _RoomsListWidgetState extends ConsumerState<RoomsListWidget> {
                               child: const Icon(Atlas.filter_thin),
                             ),
                           ),
-                        IconButton(
+                        PlusIconWidget(
```

Comment:
This, too, feels to pungent, same as the plus button on the news bar. Do we really want to draw that much attention to it? But I guess its fine for now.

Reply:
As commented above.
https://github.com/acterglobal/a3/pull/1872#discussion_r1653860069